### PR TITLE
CR-1125301 and CR-1128299

### PR DESCRIPTION
**Pull request by AShivangi**
#### Problem solved by the commit
CR-1125301 remove legacy xbutil/xbmgmt tools
CR-1128299 XRT Install - Error: unknown command 'config'

#### Bug / issue (if any) fixed, which PR introduced the bug, how it was discovered
config command is obsoleted so it needs to be replaced by the new command

#### How problem was solved, alternative solutions (if any) and why they were rejected
Since I was not able to cherry-pick from master to 2022.1, I created this PR with only the necessary changes

#### What has been tested and how, request additional testing if necessary
Locally, built and installed

### CODE REVIEW

File: src/runtime_src/core/tools/xbmgmt2/SubCmdConfigure.cpp
Comment (stsoe) [inline src/runtime_src/core/tools/xbmgmt2/SubCmdConfigure.cpp:464]:
```suggestion
```

**Review by stsoe — APPROVED** [review]
LGTM.  Two minor formatting suggestions

**Review by rozumx — APPROVED** [review]

**Comment by chvamshi-xilinx** [comment]
Hi @AShivangi , Are we waiting for any more approvals or shall i merge this change?

**Comment by AShivangi** [comment]
> Hi @AShivangi , Are we waiting for any more approvals or shall i merge this change?

Anup approved this PR, please merge it when you get a chance. Thanks

**Pull request merged**
